.env

```diff
@@ -0,0 +1,2 @@
+REACT_APP_AUTH0_DOMAIN=dev-elp-saey.us.auth0.com
+REACT_APP_AUTH0_CLIENT_ID=7qeWFCZ4wjQEC0TZ3WjBPyqxm87xsQUe
```

Comment:
this kind of stuff should never go to github!
ID,passwords etc... should always be stored inside a local .env